packages/pwa-buildpack/src/WebpackTools/PWADevServer.js

```diff
@@ -175,6 +186,33 @@ const PWADevServer = {
                 );
             }
         };
+        let devHost;
+        if (config.id) {
+            devHost = await PWADevServer.provideCustomHost(config.id);
+        } else if (config.provideUniqueHost) {
+            devHost = await PWADevServer.provideUniqueHost(
+                config.id || config.provideUniqueHost
```

Comment:
Wouldn't this statement always evaluate to `true`, given the conditions?

Reply:
Thanks, good catch.

Reply:
It's actually possibly a string. But I changed the behavior so that `config.id` and `config.provideUniqueHost` are mutually exclusive (and `config.id` is just preserving legacy behavior), so I don't need the first part of that conditional./